frontend/src/hooks/api/certificateTemplates/types.ts

```diff
@@ -0,0 +1,42 @@
+export type TCertificateTemplateListEntry = {
```

Comment:
This should ideally be just an array of `TCertificateTemplate` for response structure consistency / predictability (see endpoint comments)